src/web/index.jsp

```diff
@@ -371,14 +369,13 @@
             <a href="<%= blogFeedRSS %>" class="jive-feed-icon"><img src="images/feed-icon-16x16.gif" alt="" style="border:0;" /></a>
             <h4><fmt:message key="index.cs_blog" /></h4>
             <% long nowTime = System.currentTimeMillis();
-                if (lastBlogFeed == null || lastReleaseFeed == null || nowTime - lastRSSFetch > 21600000) {
+                if (lastBlogFeed == null || nowTime - lastRSSFetch > 21600000) {
```

Comment:
... and likewise here, an option might be useful? Obviously this works as a default.